README.md

```diff
@@ -1,29 +1,41 @@
-# vue-dynamic-property-provider
+### Vue Dynamic Property Provider
+[![Build](https://github.com/kiwiproject/vue-dynamic-property-provider/workflows/build/badge.svg)](https://github.com/kiwiproject/vue-dynamic-property-provider/actions?query=workflow%3Abuild)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=kiwiproject_vue-dynamic-property-provider&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=kiwiproject_vue-dynamic-property-provider)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=kiwiproject_vue-dynamic-property-provider&metric=coverage)](https://sonarcloud.io/summary/new_code?id=kiwiproject_vue-dynamic-property-provider)
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+![NPM Version](https://img.shields.io/npm/v/%40kiwiproject%2Fvue-dynamic-property-provider)
 
-## Project setup
-```
-npm install
+## Install
+```shell
```

Comment:
Learn something every day...I have always used `bash` but I guess this is more generic.